src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1219,7 +1250,11 @@ namespace Google.Example.Library.V1.Samples
             LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
             BookName name = new BookName("[SHELF_ID]", "[BOOK_ID]");
             Book response = await libraryServiceClient.GetBookAsync(name);
-            // FIXME: inspect the results
+            string intKeyVal = response.MapStringValue[123];
```

Comment:
I'm unclear if this is meant to be here? Or whether this due to testing code?

Reply:
Yes this is testing code. These lines are generated from: https://github.com/googleapis/gapic-generator/blob/2508ba81c8c2eea22b5a8b672f10f1c3ffb08030/src/test/java/com/google/api/codegen/testsrc/common/library_gapic.yaml#L427

Reply:
Thanks for the yaml link. I *think* this all looks OK :)